nimbus_nlp/question_classifier.py

```diff
@@ -1,16 +1,18 @@
-import spacy
+import collections
+import json
 import numpy as np
+import spacy
 import sklearn.neighbors
-from nimbus_nlp.save_and_load_model import save_model, load_latest_model, PROJECT_DIR
-import json
-from QA import db
 
+from nimbus_nlp.save_and_load_model import save_model, load_latest_model, PROJECT_DIR
+from typing import Tuple
 
 # TODO: move the functionality in this module into class(es), so that it can be more easily used as a dependency
 
 
 class QuestionClassifier:
-    def __init__(self, file_path="question_set_clean.csv"):
+    def __init__(self, db, file_path="question_set_clean.csv"):
+        self.db = db
```

Comment:
If using the DI here, need to update the other places where QuestionClassifier is constructed